src/app/pages/forms-and-validation/password/password-strength-check.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input } from '@angular/core';
+import * as Colors from '@pxblue/colors';
+
+@Component({
+    selector: 'app-password-strength-check',
+    template: `
+        <mat-list-item style="height: auto;" class="app-password-strength-check">
+            <mat-icon [style.color]="success ? Colors.blue[500] : Colors.gray[300]">{{ icon }}</mat-icon>
+            <span [style.color]="success ? Colors.gray[300] : Colors.black[500]" class="password-criteria">{{
+                validationMessage
+            }}</span>
```

Comment:
we should use the mat-typography caption class here to make sure we have all of the correct typography styles (and then add our padding.margin on top of that)